source/frontend/src/components/maps/leaflet/LayerPopup/MultiplePropertyPopupView.tsx

```diff
@@ -41,21 +44,27 @@ export const MultiplePropertyPopupView: React.FC<
     }
   };
 
-  const propertyProjections =
-    featureDataset?.parcelFeatures
-      ?.map<PropertyProjection>(x => {
-        return {
-          pid: x.properties.PID_FORMATTED,
-          isStrataLot: isStrataLot(x),
-          feature: x,
-        };
-      })
-      .sort((a, b) => {
-        if (a.isStrataLot === b.isStrataLot) return 0;
-        if (a.isStrataLot) return -1;
-        if (b.isStrataLot) return 1;
-        return 0;
-      }) ?? [];
+  const groupedFeatures = chain(featureDataset?.parcelFeatures)
```

Comment:
Might need a comment here for the grouping->ungrouping mechanism.

Question, if there are multiple plan numbers, wouldn't this result in some strata lots not being sorted to the top? That is a functional change - not sure if that is what was desired. Is the desire here to show that x properties all have the same plan? If so, maybe the plan should be displayed in addition to the pid/pin.

As an example, I think this change would result in something like:
Common Property VIS8102
125-521-123
125-521-122
125-521-121
125-521-121
Common Property VIS56215
512-516-231

whereas previously this would have displayed (without the grouped sort):
Common Property VIS8102
Common Property VIS56215
125-521-123
125-521-122
125-521-121
125-521-121
512-516-231

Reply:
Yes, that's correct. I was surprised by a single location having multiple Common Properties. I figured that showing the properties in a grouped manner makes more sense than just putting the common strata on top.